noodles-editor/src/noodles/noodles.tsx

```diff
@@ -383,52 +405,62 @@ export function getNoodles(): Visualization {
   // biome-ignore lint/correctness/useExhaustiveDependencies: loadProjectFile would cause infinite loop
   useEffect(() => {
     ;(async () => {
-      if (projectId) {
-        // First try to load from static files (for built-in examples)
+      // If no currentProjectId, load the default new project
+      if (!currentProjectId) {
         try {
-          const req = await fetch(`./noodles/${projectId}/noodles.json`)
-          const noodlesFile = (await req.json()) as Partial<NoodlesProjectJSON>
-          const project = await migrateProject({
-            ...EMPTY_PROJECT,
-            ...noodlesFile,
-          } as NoodlesProjectJSON)
-          // Set project name and storage type for public projects so @/ asset paths work
-          setCurrentDirectory(null, projectId)
-          setActiveStorageType('publicFolder')
-          loadProjectFile(project, projectId)
+          const project = await fetch(newProjectJSON).then(res => res.json()) as NoodlesProjectJSON
+          loadProjectFile(project)
```

Comment:
Resurfacing an existing limitation to polish _in a future PR_, just since it's relevant here too:

Data importing only currently functions when a project is using the OPFS or native file system storage type, and is in the state where that storage is granted permission. FileOp and drag-n-drop features would ideally be aware of this, and we'd have UX to establish a "saved" project when attempting to import